examples/tutorials/Advanced/Federated LSTM-Predicting Next Word.ipynb

```diff
@@ -0,0 +1,520 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<h1>Federated LSTM-Predicting Next Word</h1>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This example demonstrates how predicting next word model in PyTorch could be trained using federated learning with PySyft. We distribute the text data to two workers Bob and Alice to whom the model is sent and trained. Upon training the model the trained model is sent back to the owner of the model and used to make predictions.Federated learning applied to predicting next word helps to improve accuracy of predictor while ensuring the exact text of the end users are not known. \n",
```

Comment:
Maybe add a link to the Pytorch only tutorial so the reader can take a look if they want to

Reply:
.Federated learning applied ->. Federated learning applied (missing space)